src/atoms/walletLabel/types.ts

```diff
@@ -0,0 +1,10 @@
+import { HeaderSizes, BodySizes } from '../texts/types';
+
+export type WalletLabelProps = {
```

Comment:
if you use type instead of interface for shortenText you can use union type here so you wont have to always add the types in two places in case shorten text type gets modified